cmd/genkey.go

```diff
@@ -142,6 +162,15 @@ func generateAuthKeys(guid efiutil.EFIGUID, keyPath, keyType string) error {
 		sigdb.AppendDatabase(oemSigDb)
 	}
 
+	// TODO: PK too?
```

Comment:
I'm not sure if we can enroll the "custom" (exported) PK instead of ours. I'm thinking that maybe it won't allow us to enroll  the KEK one then which will be signed by a key not maching the one signing the PK. I'd rather leave PK out for now and enroll ours, since that's the scenario I've tested.

Reply:
isnt PK signs KEK which in turn signs DB? in that case we may need the PK to always be the user one so we can sign all this stuff

Reply:
yes, that's what I was thinking as well. I'll skip the PK then form custom certs. We'll use ours.